broker/src/main/java/io/camunda/zeebe/broker/logstreams/LogDeletionService.java

```diff
@@ -35,14 +36,12 @@ public String getName() {
 
   @Override
   protected void onActorStarting() {
-    persistedSnapshotStore.addSnapshotListener(this);
```

Comment:
Wouldn't this mean we are triggered twice now?

Reply:
No, the snapshot store uses a set, so we will be triggered once.

Reply:
thanks